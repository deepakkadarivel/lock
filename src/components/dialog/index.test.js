import React from 'react';
import { create } from 'react-test-renderer';
import Dialog from './index';

describe('Dialog component', () => {
    test('Renders Dialog component', () => {
        const component = create(
            <Dialog
                header="test form header"
                form={
                    <form onSubmit={() => {}}>
                        <input type="text" name="name" />
                    </form>
                }
            />
        );
        expect(component.toJSON()).toMatchSnapshot();
    });
});
