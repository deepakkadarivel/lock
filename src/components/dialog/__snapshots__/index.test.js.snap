// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dialog component Renders Dialog component 1`] = `
<div
  className="Dialog"
>
  <h2>
    test form header
  </h2>
  <form
    onSubmit={[Function]}
  >
    <input
      name="name"
      type="text"
    />
  </form>
</div>
`;
